### PR TITLE
Create vtex-shipping-network-labels-integration.md

### PR DESCRIPTION
**Pull request by Isabella-Veloso**
#### Types of changes
- [x] New content (guides, endpoints, app documentation)
- [ ] Improvement (make a documentation even better)
- [ ] Fix (fix a documentation error)
- [ ] Spelling and grammar accuracy (self-explanatory)

Documentation for VTEX Shipping Network labels integration for the closed beta. It will be shared with only a couple of clients for feedback and iteration. The documentation was created by the team, and I have made a basic review.

### CODE REVIEW

File: docs/guides/Integration-Guides/vtex-shipping-network/vtex-shipping-network-labels-integration.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+## Overview
+
+The VTEX Shipping Network module is responsible for integrating your store with carriers via a defined protocol. The integration is divided into three workflows:
+
+1. Notification
+2. Tracking
+3. Shipping label creation
+
+The following lifecycle overview of a Package created in the VTEX Shipping Network module (SPN) will provide an overview of these workflows:
```
Comment (jumeyermattosarruda) [inline docs/guides/Integration-Guides/vtex-shipping-network/vtex-shipping-network-labels-integration.md:19]:
```suggestion
 A Package created in the VTEX Shipping Network module (SPN) has the following lifecycle, including these workflows:
```

File: docs/guides/Integration-Guides/vtex-shipping-network/vtex-shipping-network-labels-integration.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+1. The cycle begins when an order is invoiced in the OMS module, which results in SPN creating all of its packages. The new packages will have an `Invoiced` [status](https://help.vtex.com/en/tutorial/order-flow-and-status--tutorials_196).
+2. The Carrier App will then receive a notification request for all packages, and their status will move to `carrier_notified`.
```
Comment (jumeyermattosarruda) [inline docs/guides/Integration-Guides/vtex-shipping-network/vtex-shipping-network-labels-integration.md:22]: Dá pra adicionar o link do Carrier App aqui. Tá publicada a doc?

File: docs/guides/Integration-Guides/vtex-shipping-network/vtex-shipping-network-labels-integration.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+The Package lifecycle ends when it is delivered.
+
+## Integration
+
+The endpoint described in this document is responsible for creating the Shipping Labels of all the packages associated with an order. Given an `orderId`, it will provide an URL that when accessed downloads a pdf file containing the Shipping Labels.
+
+>❗ The URL returned by the API is valid for 5 days from the moment it was requested, and expires after that period. If the file is required after the link has expired, only a new request with the same `orderId` will provide a new valid URL.
+
+Be aware that VTEX Shipping Network only keeps track of Fulfillment orders, therefore, the `orderId` used on the endpoint must have their origin as Fulfillment. All Fulfillment Orders contain a prefix, for example:
```
Comment (jumeyermattosarruda) [inline docs/guides/Integration-Guides/vtex-shipping-network/vtex-shipping-network-labels-integration.md:34]: Temos link pra explicar mais sobre o que são fulfillment orders?

Comment (jumeyermattosarruda) [reply docs/guides/Integration-Guides/vtex-shipping-network/vtex-shipping-network-labels-integration.md:34]: Tinha pensado em algo como [aqui](https://developers.vtex.com/docs/guides/multilevel-omnichannel-inventory#order-flow), mas não tem a terminologia de Fulfillment orders nessa doc de MOI

**Review by jumeyermattosarruda — CHANGES_REQUESTED** [review]

File: docs/guides/Integration-Guides/vtex-shipping-network/vtex-shipping-network-labels-integration.md (hunk 3 above)
Comment (Isabella-Veloso) [reply docs/guides/Integration-Guides/vtex-shipping-network/vtex-shipping-network-labels-integration.md]: Feito.

**Pull request merged**